packages/transforms/mock/test/mocking.spec.ts

```diff
@@ -79,7 +79,6 @@ describe('mocking', () => {
 
     const users = result.data?.users;
     expect(users).toBeTruthy();
-    expect(users).toHaveLength(1);
```

Comment:
Why are there more then 1 user?

Reply:
When the return type is an array, 2 mock results are generated in `@graphql-tools/mock`.